app/scripts/directives/oscFileInput.js

```diff
@@ -127,6 +127,10 @@ angular.module('openshiftConsole')
             if (files.length > 0 ) {
               scope.file = _.head(files);
               addFile(scope.file);
+            } else {
+              scope.uploadError = true;
+              scope.cleanInputValues();
+              Logger.error("Could not read file", e);
```

Comment:
This change looks unrelated

Reply:
Correct, cherry picked jakub's PR for the file input, should have saw before my commit, removing.